metricbeat/docker-compose.yml

```diff
@@ -134,6 +134,19 @@ services:
 
   mongodb:
     build: ./module/mongodb/_meta
+    command: mongod --replSet beats
+    ports:
+      - "27017:27017"
+
+  mongodbreplica:
+    build:
+      context: ./module/mongodb/_meta
+      dockerfile: Dockerfile.replicasetup
+    links:
+      - "mongodb:mongodb"
+    depends_on:
+      - mongodb
+    entrypoint: [ "bash", "/replicaset.sh" ]
```

Comment:
What about having an only `mongodb` service with the replicaset enabled? Would it affect other metrics?